lib/phoenix_swagger/plug/swaggerui.ex

```diff
@@ -22,10 +22,10 @@ defmodule PhoenixSwagger.Plug.SwaggerUI do
   alias Plug.Conn
 
   # Serve static assets before routing
-  plug Plug.Static, at: "/", from: :phoenix_swagger
+  plug(Plug.Static, at: "/", from: :phoenix_swagger)
```

Comment:
`plug` macro is usually whitelisted as no parens required (https://github.com/elixir-plug/plug/blob/master/.formatter.exs#L3-L4)